frontend/server/src/Controllers/School.php

```diff
@@ -237,24 +273,11 @@ public static function apiSchoolsOfTheMonth(\OmegaUp\Request $r) {
         $firstDayOfNextMonth = $currentDate->modify('first day of next month');
         $date = $firstDayOfNextMonth->format('Y-m-d');
 
-        /** @var list<array{school_id: int, name: string, country_id: string, score: float}> */
-        $rank = \OmegaUp\Cache::getFromCacheOrSet(
-            \OmegaUp\Cache::SCHOOLS_OF_THE_MONTH,
-            "{$date}-{$rowcount}",
-            function () use (
-                $date,
-                $rowcount
-            ): array {
-                return \OmegaUp\DAO\SchoolOfTheMonth::calculateSchoolsOfMonthByGivenDate(
-                    $date,
-                    $rowcount
-                );
-            },
-            60 * 60 * 12 // 12 hours
-        );
-
         return [
-            'rank' => $rank,
+            'rank' => self::getTopSchoolsOfTheMonth(
+                $rowcount,
+                true /** canUseCache */
```

Comment:
nit:

```suggestion
                /*$canUseCache=*/true
```

los comentarios de la forma `/** ... */` se usan únicamente para docstrings de métodos y anotaciones de psalm.